tests/Trakx.CoinGecko.ApiClient.Tests/Unit/CoinGeckoClientTests.cs

```diff
@@ -1,4 +1,5 @@
 using Microsoft.Extensions.Caching.Memory;
+using Trakx.Common.ApiClient.Extensions;
 using Trakx.Common.Testing.Mocks;
 
 namespace Trakx.CoinGecko.ApiClient.Tests.Unit;
```

Comment:
The diff here is messed up.
It's not a comparison at all.

The tests were moved to `CoinGeckoPricesClientTests`.
The code from the "internal" file was moved here.